doc/api/buffer.md

```diff
@@ -3820,6 +3820,40 @@ for (const value of buf) {
 //   114
 ```
 
+### `buf.toInt([radix])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `radix` {number} An integer between `2` and `36` that represents the radix
+  (the base in mathematical numeral systems) of the `string`.
+
+Converts a valid `buf` containing items in the range of 0x48 (1) to 0x57 (9)
```

Comment:
```suggestion
Converts a valid `buf` containing items in the range of 0x48 (0) to 0x57 (9)
```

It looks a bit counter-intuitive to me. Why are digits limited to 0-9 while we can have a radix up to 36?

Reply:
Would it be better if I update the documentation to indicate that items are ASCII numbers?